app/src/main/java/org/simple/clinic/drugs/selection/entry/CustomPrescriptionEntryUpdate.kt

```diff
@@ -25,19 +25,12 @@ class CustomPrescriptionEntryUpdate : Update<CustomPrescriptionEntryModel, Custo
   }
 
   private fun onPrescriptionFetched(
-      model: CustomPrescriptionEntryModel,
       prescription: PrescribedDrug
   ): Next<CustomPrescriptionEntryModel, CustomPrescriptionEntryEffect> {
     return if (prescription.isDeleted) {
       dispatch(CloseSheet)
     } else {
-      var updatedModel = model
-          .drugNameChanged(prescription.name)
-
-      if (prescription.dosage != null)
-        updatedModel = updatedModel.dosageChanged(prescription.dosage)
-
-      next(updatedModel, SetMedicineName(prescription.name), SetDosage(prescription.dosage))
+      dispatch(SetMedicineName(prescription.name), SetDosage(prescription.dosage))
```

Comment:
Not necessary to fix this, but this change is unrelated to the commit message. Was it meant to be a separate commit?

Reply:
This was breaking the tests. I realized that this change was not necessary although I was confused about it for some time. We would display the fetched prescription in the UI but if we do not update the model to reflect the current value of `drugName` and `dosage`, will that be an unexpected UI state?
Rn, we only update the `drugName` and `dosage` when there is `TextChanged` callback.

Reply:
Ah, alright.